README.md

```diff
@@ -35,51 +35,60 @@ opam pin add coq 8.10.2
 opam pin add coq-serapi 8.10.0+0.7.1
 ```
 
-### Installation of Python and PyTorch
+### Installation of PyTorch and Python libraries
 
-We recommend installing the required versions of Python and PyTorch
+We strongly recommend installing the required versions of Python and PyTorch
 using [Conda](https://docs.conda.io/en/latest/miniconda.html).
 
-Then, install PyTorch by following the instructions
-[here](https://pytorch.org/get-started/previous-versions/#v110), using
-the installation command suitable for your operating system, Python
-package manager, and whether you want to use it on a CPU or GPU.
-
-Next, either download a Roosterize [release][releases-link]
-or clone the Roosterize repository and enter the directory:
+First, create a Python 3.7 environment using Conda:
 ```
-git clone https://github.com/EngineeringSoftware/roosterize.git
-cd roosterize
+conda create --name roosterize python=3.7 pip
+conda activate roosterize
 ```
 
-To install other required Python libraries (`pip` is included with
-Python installation):
+Then, install PyTorch 1.1.0 by following the
+[official instructions](https://pytorch.org/get-started/previous-versions/#conda-5),
+using the command suitable for your operating system and whether you want to use
+it on a CPU or GPU. For example, the following command installs PyTorch for
+CPU-only use on Linux or Windows:
+```
+conda install pytorch-cpu==1.1.0 torchvision-cpu==0.3.0 cpuonly -c pytorch
+```
+Finally, install the required Python libraries:
 ```
-pip install -r requirements.txt
+conda install configargparse nltk future seutil==0.4.12 six torchtext==0.4.0 tqdm==4.30.*
 ```
 
-[releases-link]: https://github.com/EngineeringSoftware/roosterize/releases
+### Installation of Roosterize and trained models
 
-### Download pre-trained models
+To install Roosterize itself, clone the GitHub repository and enter the root directory:
+```
+git clone https://github.com/EngineeringSoftware/roosterize.git
+cd roosterize
+```
 
-We have trained several models using our [corpus][math-comp-corpus], which
-follows the lemma naming conventions used in the [Mathematical Components][math-comp-website]
-family of projects. The pre-trained models are available for separate [download][models-link].
+Next, you need to obtain pre-trained models that capture
+naming conventions. We have trained several models using our
+[corpus][math-comp-corpus], which follows the conventions
+used in the [Mathematical Components][math-comp-website]
+family of projects. These models are available for separate
+[download][models-link].
 
-Put the archive with the models (`models.tgz`) in the Roosterize root
-directory, and then uncompress it:
+To use our pre-trained models, go to the Roosterize
+root directory, and download and unpack the archive (`models.tgz`):
 ```
+wget https://github.com/EngineeringSoftware/roosterize/releases/download/v8.10.0/models.tgz
 tar xzf models.tgz
 ```
```

Comment:
This part (download pre-trained models) will be replaced with the new download_global_model API.

Reply:
I see, this is good to know, but maybe this is OK to merge in the meantime? Or should I remove it?

Reply:
I'll handle the merge of this part.